modules/configuring-external-databases/proc-configuring-postgresql-instance-using-helm.adoc

```diff
@@ -76,7 +76,7 @@ EOF
 
 . Configure your PostgreSQL instance in the Helm configuration file named `values.yaml`:
 +
-[source,yaml]
+[source,yaml, subs="+attributes"]
```

Comment:
To tell asciidoctore that `{` are really `{` and not the beginning of an attribute name, escape the first `{` when defining environment variables, such as: `$\{POSTGRES_HOST}`.